### PR TITLE
some mappings for worldrenderer

### PR DESCRIPTION
**Pull request by woodiertexas**

### CODE REVIEW

File: mappings/net/minecraft/unmapped/ChunkBuilder.mapping
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/unmapped/ChunkBuilder.mapping:1]: please put this in a package

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]
Since all the `matrix4f` params are along side other `Martrix4f` params, they should have more descriptive names.

File: mappings/net/minecraft/unmapped/ChunkBuilder.mapping [hunk 1]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_jnliocmw net/minecraft/unmapped/ChunkBuilder
```
Comment (OroArmor) [inline mappings/net/minecraft/unmapped/ChunkBuilder.mapping]: This isnt really a chunk builder. Its similar but different enough this name is misleading. I would leave this unmapped for now

File: mappings/net/minecraft/client/render/WorldRenderer.mapping [hunk 2]
```diff
@@ -18,6 +18,7 @@ CLASS net/minecraft/unmapped/C_sfkkabhx net/minecraft/client/render/WorldRendere
 	FIELD f_ivihlszs SNOW Lnet/minecraft/unmapped/C_ncpywfca;
 	FIELD f_izxsabgg MOON_PHASES Lnet/minecraft/unmapped/C_ncpywfca;
 	FIELD f_jpospzcg MINIMUM_FOG_DISTANCE I
+	FIELD f_kfsuutjj chunkBuilder Lnet/minecraft/unmapped/C_jnliocmw;
```
Comment (OroArmor) [inline mappings/net/minecraft/client/render/WorldRenderer.mapping]: see below comment

**Review by OroArmor — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/render/WorldRenderer.mapping [hunk 3]
```diff
@@ -30,6 +30,7 @@ CLASS net/minecraft/unmapped/C_sfkkabhx net/minecraft/client/render/WorldRendere
 	FIELD f_ntktcmew rainSizeX [F
 	FIELD f_ntzttglo TRANSPARENT_SORT_AMOUNT I
 	FIELD f_ofkpyvlk DIRECTIONS [Lnet/minecraft/unmapped/C_xpuuihxf;
+	FIELD f_oppequxe blockSounds Ljava/util/Map;
```
Comment (OroArmor) [inline mappings/net/minecraft/client/render/WorldRenderer.mapping]:
```suggestion
	FIELD f_oppequxe currentSounds Ljava/util/Map;
```

File: mappings/net/minecraft/client/render/WorldRenderer.mapping [hunk 4]
```diff
@@ -240,10 +241,13 @@ CLASS net/minecraft/unmapped/C_sfkkabhx net/minecraft/client/render/WorldRendere
 	METHOD m_khytiwdn setupFrustum (Lnet/minecraft/unmapped/C_vgpupfxx;Lorg/joml/Matrix4f;Lorg/joml/Matrix4f;)V
 		ARG 1 pos
 		ARG 2 projectionMatrix
+		ARG 3 matrix4f
 	METHOD m_kkvlftbz addVertexFromFrustum (Lnet/minecraft/unmapped/C_igthdzux;Lorg/joml/Matrix4f;I)V
 		ARG 1 consumer
 		ARG 2 matrix
 		ARG 3 frustumIndex
+	METHOD m_kvtxvfgf createParticles (ILnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_rlomrsco;Lnet/minecraft/unmapped/C_lwmufxcy;)V
+		ARG 4 defaultParticleType
```
Comment (OroArmor) [inline mappings/net/minecraft/client/render/WorldRenderer.mapping]:
```suggestion
		ARG 4 particle
```

**Review by OroArmor — CHANGES_REQUESTED** [review]
the `matrix4f` parameters are probably the `viewMatrix`. HOWEVER, i would recheck that something didnt change because the order is usually model, view, then projection, but it seems that projection is before view, which is odd.

**Comment by woodiertexas** [comment]
> the `matrix4f` parameters are probably the `viewMatrix`. HOWEVER, i would recheck that something didnt change because the order is usually model, view, then projection, but it seems that projection is before view, which is odd.

yea, they were `matrix4f` when i was cross-referencing `WorldRenderer` in enigma with `WorldRenderer.java` in the intelliJ planetarium project

**Comment by woodiertexas** [comment]
> the `matrix4f` parameters are probably the `viewMatrix`. HOWEVER, i would recheck that something didnt change because the order is usually model, view, then projection, but it seems that projection is before view, which is odd.

in enigma:
java_AVFFJSPAU1

in the planetarium project:
idea64_JGZThcg3bH

**Review by eristhea — CHANGES_REQUESTED** [review]
looks good; besides the `matrix4f` parameters

**Comment by woodiertexas** [comment]
> looks good; besides the `matrix4f` parameters

so what name would you give those parameters?

File: mappings/net/minecraft/client/render/WorldRenderer.mapping [hunk 5]
```diff
@@ -240,10 +241,13 @@ CLASS net/minecraft/unmapped/C_sfkkabhx net/minecraft/client/render/WorldRendere
 	METHOD m_khytiwdn setupFrustum (Lnet/minecraft/unmapped/C_vgpupfxx;Lorg/joml/Matrix4f;Lorg/joml/Matrix4f;)V
 		ARG 1 pos
 		ARG 2 projectionMatrix
+		ARG 3 matrix4f
```
Comment (OroArmor) [inline mappings/net/minecraft/client/render/WorldRenderer.mapping]:
```suggestion
		ARG 2 modelViewMatrix
		ARG 3 projectionMatrix
```

File: mappings/net/minecraft/client/render/WorldRenderer.mapping [hunk 6]
```diff
@@ -270,10 +274,13 @@ CLASS net/minecraft/unmapped/C_sfkkabhx net/minecraft/client/render/WorldRendere
 		ARG 4 gameRenderer
 		ARG 5 lightmapTextureManager
 		ARG 6 projectionMatrix
+		ARG 7 matrix4f
```
Comment (OroArmor) [inline mappings/net/minecraft/client/render/WorldRenderer.mapping]:
```suggestion
		ARG 6 modelViewMatrix
		ARG 7 projectionMatrix
```

File: mappings/net/minecraft/client/render/WorldRenderer.mapping [hunk 7]
```diff
@@ -305,6 +313,7 @@ CLASS net/minecraft/unmapped/C_sfkkabhx net/minecraft/client/render/WorldRendere
 		ARG 1 camera
 	METHOD m_qwqxlbnd renderSky (Lorg/joml/Matrix4f;Lorg/joml/Matrix4f;FLnet/minecraft/unmapped/C_dcmtgrvm;ZLjava/lang/Runnable;)V
 		ARG 1 projectionMatrix
+		ARG 2 matrix4f
```
Comment (OroArmor) [inline mappings/net/minecraft/client/render/WorldRenderer.mapping]:
```suggestion
		ARG 1 modelViewMatrix
		ARG 2 projectionMatrix
```

**Review by OroArmor — CHANGES_REQUESTED** [review]
Confirmed by double checking the `Frustrum` constructor mapping

**Review by OroArmor — APPROVED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/client/render/WorldRenderer.mapping [hunk 8]
```diff
@@ -405,6 +419,7 @@ CLASS net/minecraft/unmapped/C_sfkkabhx net/minecraft/client/render/WorldRendere
 		ARG 4 y
 		ARG 6 z
 		ARG 8 color
+	METHOD m_znnjtopn createChunkBuilderDebugLine ()Ljava/lang/String;
```
Comment (OroArmor) [inline mappings/net/minecraft/client/render/WorldRenderer.mapping]:
```suggestion
	METHOD m_znnjtopn getChunkBuilderDebugString ()Ljava/lang/String;
```
I know I said the previous thing, but I want to match the other functions like this.

File: mappings/net/minecraft/util/math/MathHelper.mapping [hunk 9]
```diff
@@ -274,6 +274,7 @@ CLASS net/minecraft/unmapped/C_zpccxxiz net/minecraft/util/math/MathHelper
 	METHOD m_rzxxkxfg lengthSquared (DD)D
 		ARG 0 a
 		ARG 2 b
+	METHOD m_saszmoqp magnitude (FFF)F
```
Comment (OroArmor) [inline mappings/net/minecraft/util/math/MathHelper.mapping]:
```suggestion
	METHOD m_saszmoqp lengthSquared (FFF)F
```

**Review by OroArmor — CHANGES_REQUESTED** [review]

**Review by OroArmor — APPROVED** [review]

**Review by eristhea — APPROVED** [review]

**Pull request merged**